src/js/components/Button/__tests__/Button-kind-test.js

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+
+import 'jest-styled-components';
+
+import { cleanup, render } from '@testing-library/react';
+import { grommet } from '../../../themes/grommet';
+import { Grommet, Button } from '../..';
+
+describe('Button kind', () => {
+  afterEach(cleanup);
+
+  test('grommet theme', () => {
+    const { container } = render(
+      <Grommet theme={grommet}>
```

Comment:
As discussed, let's use the button form hpe theme https://github.com/grommet/grommet-theme-hpe/blob/a2d7bbd8832f37c03e415a6cc6ec85178966a0bb/src/js/index.js#L243 instead of grommet theme.